### PR TITLE
Update Keycloak module version

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update Keycloak module version
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| Sponsor company   | ~
| How to test?      | ~

### CODE REVIEW

**Review by nesrineabdmouleh — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
